build-system/tasks/extension-helpers.js

```diff
@@ -507,6 +511,46 @@ async function buildExtension(
   await buildExtensionJs(extDir, name, version, latestVersion, options);
 }
 
+/** @type {TransformCache} */
+let cssCache;
+
+/**
+ * Writes an extensions's CSS to its npm dist folder.
+ *
+ * @param {string} extDir
+ * @param {Object} options
+ * @return {Promise<void>}
+ */
+async function buildNpmCss(extDir, options) {
+  const startCssTime = Date.now();
+  const jssFile = (await globby(path.join(extDir, '**', '*.jss.js')))[0];
+  if (!jssFile) {
+    return;
+  }
+
+  // Lazily instantiate the TransformCache
+  if (!cssCache) {
+    cssCache = new TransformCache('jss-cache', '.css');
```

Comment:
Important: I think the directory name should start with a period, right?

```suggestion
    cssCache = new TransformCache('.jss-cache', '.css');
```

Reply:
Definitely! DOne